specification/saas/resource-manager/Microsoft.SaaS/preview/2018-03-01-beta/saas.json

```diff
@@ -356,6 +430,483 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SaaS/resources/{resourceName}": {
```

Comment:
>subscriptions [](start = 6, length = 13)

You need to have an API to GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SaaS/resources.

If this is a tracked resource, you also need an API to list by subscription as well. Both of these got flagged by the linter.